packages/cli/src/zosjobs/list/jobs/Jobs.handler.ts

```diff
@@ -32,17 +32,28 @@ export default class JobsHandler extends ZosmfBaseHandler {
         // Obtain the list of jobs - by default uses the session user and * for owner and prefix.
         const owner: string = (params.arguments.owner != null) ? params.arguments.owner : null;
         const prefix: string = (params.arguments.prefix != null) ? params.arguments.prefix : JobsConstants.DEFAULT_PREFIX;
-        const jobs: IJob[] = await GetJobs.getJobsCommon(this.mSession, {owner, prefix});
+        const execdata: boolean = params.arguments["exec-data"];
```

Comment:
`yargs` makes option names accessible both in kebab and camel case form. So `params.arguments.execData` should also work without any quotation marks or brackets necessary.